src/e2e-test/java/io/cdap/plugin/common/stepsdesign/TestSetupHooks.java

```diff
@@ -201,7 +206,8 @@ public static void setTempTargetGCSBucketName() {
     gcsTargetBucketName = "cdf-e2e-test-" + UUID.randomUUID();
     PluginPropertyUtils.addPluginProp("gcsTargetBucketName", gcsTargetBucketName);
     PluginPropertyUtils.addPluginProp("gcsTargetPath", "gs://" + gcsTargetBucketName);
-    BeforeActions.scenario.write("GCS target bucket name - " + gcsTargetBucketName);
+    System.out.println(gcsTargetBucketName);
```

Comment:
Remove the print statement.

Reply:
done
commit id- https://github.com/cloudsufi/google-cloud/commit/defc54df110255ef539dea8d206481cc145b2fec